lib/resqued/worker.rb

```diff
@@ -80,7 +83,7 @@ def try_start
         Resqued::Listener::ALL_SIGNALS.each { |signal| trap(signal, 'DEFAULT') }
         trap(:QUIT) { exit! 0 } # If we get a QUIT during boot, just spin back down.
         $0 = "STARTING RESQUE FOR #{queues.join(',')}"
-        resque_worker = Resque::Worker.new(*queues)
+        resque_worker = @worker_factory.call(queues)
         resque_worker.term_child = true if resque_worker.respond_to?('term_child=')
         Resque.redis.client.reconnect
```

Comment:
Would it make sense to move these two lines into the default worker factory, too?

Reply:
Makes sense to me! Updated 👍